sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -108,16 +120,13 @@ def create_experiment(
             raise ValueError("Experiment must have a name or generateName")
 
         try:
-            outputs = self.custom_api.create_namespaced_custom_object(
+            self.custom_api.create_namespaced_custom_object(
                 constants.KUBEFLOW_GROUP,
                 constants.KATIB_VERSION,
                 namespace,
                 constants.EXPERIMENT_PLURAL,
                 experiment,
             )
-            experiment_name = outputs["metadata"][
-                "name"
-            ]  # if "generate_name" is used, "name" gets a prefix from server
```

Comment:
We need to keep that in case experiment is created using generate_name

Reply:
Got it. My understanding of the code was not complete hence I removed it incorrectly. I have reverted this code change and updated the `create_namespaced_custom_object_response` function to account for the output of the `create_namespaced_custom_object` function.